tests/qunit/fixtures/wp-api-generated.js

```diff
@@ -4632,6 +4632,46 @@ mockedApiResponse.Schema = {
                 }
             ]
         },
+        "/wp/v2/block-renderer/(?P<name>core/navigation)": {
```

Comment:
@TimothyBJacobs or @kadamwhite, do you still plan to work on improvements for `block-render` endpoint? It's too granular, the definition it's nearly identical for all registered blocks. In WordPress 5.5, we want to enable server-side registration for all core blocks. This object might become a few times as big as it is now.

Reply:
Definitely. Will make sure it is an early priority for 5.5.